README.md

```diff
@@ -123,12 +138,6 @@ Make sure to update your packages:
 pkg up
 ```
 
-In the case mpv only plays audio, you can try running this command:
-```sh
-echo 'am start --user 0 -a android.intent.action.VIEW -d "$1" -n is.xyz.mpv/.MPVActivity' > $PREFIX/bin/mpv
-```
```

Comment:
don't delete this 😔

Reply:
wait, i did? wtf

Reply:
ah! i just copied over my readme from V2; but this part was added in V3